src/time_zone_impl.cc

```diff
@@ -31,9 +31,12 @@ using TimeZoneImplByName =
 TimeZoneImplByName* time_zone_map = nullptr;
 
 // Mutual exclusion for time_zone_map.
-// This mutex is intentionally "leaked" to avoid the static deinitialization
-// order fiasco (std::mutex's destructor is not trivial on many platforms).
-std::mutex* time_zone_mutex = new std::mutex;
+std::mutex& TimeZoneMutex() {
+  // This mutex is intentionally "leaked" to avoid the static deinitialization
+  // order fiasco (std::mutex's destructor is not trivial on many platforms).
+  static std::mutex *time_zone_mutex = new std::mutex;
```

Comment:
Shift the `*` from the variable name to the type name.

Reply:
Whoops! Fixed, thanks.